index.html

```diff
@@ -25,9 +24,9 @@ <h1>My Awesome Books</h1>
       </section>
     <form id="form" action="" class="add-book-form">
         <div>
-            <h2>Add new book</h2>
+            <h2>Add a new book</h2>
             <div id="form-div">
-                <input type="text" id="title" class="title-book" name="title" placeholder="Title"><br>
+                <input type="text" id="title" class="title-book" name="title" placeholder="Title"></br>
                 <input type="text" id="author" class="author-book" name="author" placeholder="Author"><br>
                 <button type="submit" value="submit" class="add-btn">Add</button> 
```

Comment:
- [Optional] Great Job for adding the `add` and `remove` functionality, I think it would be better if you clear the input fields after successfully adding the book to improve the user interactivity 👍